src/App/Controller/SecurityController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
+
+
+class SecurityController extends Controller
+{
+    public function login(Request $request, AuthenticationUtils $authUtils)
```

Comment:
You should type hint return to `Response`
`public function login(Request $request, AuthenticationUtils $authUtils) : Response`